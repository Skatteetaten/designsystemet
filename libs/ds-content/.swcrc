{
  "jsc": {
    "target": "es2021",
    "parser": {
      "syntax": "typescript",
      "tsx": true,
      "decorators": true,
      "dynamicImport": true
    },
    "transform": {
      "react": {
        "runtime": "automatic",
        "importSource": "react"
      },
      "decoratorMetadata": true,
      "legacyDecorator": true
    },
    "keepClassNames": true,
    "externalHelpers": true,
    "loose": true
  },
  "module": {
    "type": "es6"
  },
  "sourceMaps": true,
  "exclude": ["jest.config.ts",".*\\.spec.tsx?$",".*\\.test.tsx?$","./src/jest-setup.ts$","./**/jest-setup.ts$",".*.js$"]
}
