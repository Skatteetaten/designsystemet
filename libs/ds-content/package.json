{
  "name": "@skatteetaten/ds-content",
  "groupId": "no.skatteetaten.aurora",
  "version": "1.4.0",
  "license": "Apache-2.0",
  "homepage": "https://www.skatteetaten.no/stilogtone/designsystemet/",
  "type": "module",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Skatteetaten/designsystemet.git"
  },
  "publishConfig": {
    "registry": "https://nexus.sits.no/repository/npm-internal/"
  },
  "peerDependencies": {
    "react": "^18",
    "@skatteetaten/ds-core-utils": "1.4.0",
    "@skatteetaten/ds-typography": "1.4.0",
    "@skatteetaten/ds-icons": "1.4.0",
    "react-i18next": "^14",
    "i18next": "^23"
  },
  "scripts": {},
  "dependencies": {},
  "devDependencies": {}
}
