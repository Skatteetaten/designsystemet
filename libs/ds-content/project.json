{
  "name": "ds-content",
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/ds-content/src",
  "projectType": "library",
  "tags": [],
  "implicitDependencies": ["!ds-dev-config"],
  "targets": {
    "lint": {
      "executor": "@nx/eslint:lint",
      "outputs": ["{options.outputFile}"]
    },
    "build": {
      "executor": "@nx/rollup:rollup",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/ds-content",
        "tsConfig": "libs/ds-content/tsconfig.lib.json",
        "project": "libs/ds-content/package.json",
        "entryFile": "libs/ds-content/src/index.ts",
        "external": ["react/jsx-runtime"],
        "rollupConfig": ["libs/ds-content/rollup.config.js"],
        "compiler": "swc",
        "assets": [
          {
            "glob": "libs/ds-content/README.md",
            "input": ".",
            "output": "."
          },
          {
            "glob": "LICENSE",
            "input": ".",
            "output": "."
          }
        ],
        "babelUpwardRootMode": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/ds-content/jest.config.ts"
      }
    },
    "stylelint": {
      "executor": "nx-stylelint:lint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": [
          "libs/ds-content/**/*.css",
          "libs/ds-content/**/*.scss"
        ]
      }
    },
    "tsc": {
      "executor": "@skatteetaten/ds-dev-config:tsCheck",
      "options": {
        "tsConfig": ["tsconfig.lib.json", "tsconfig.spec.json"]
      },
      "dependsOn": [
        {
          "projects": "ds-core-designtokens",
          "target": "build"
        }
      ]
    },
    "set-version": {
      "executor": "nx:run-commands",
      "options": {
        "command": "npm version {args.packageVersion} --no-git-tag",
        "cwd": "libs/ds-content"
      }
    },
    "publish": {
      "executor": "nx:run-commands",
      "options": {
        "command": "npm publish dist/libs/ds-content {args.switches}"
      }
    }
  }
}
