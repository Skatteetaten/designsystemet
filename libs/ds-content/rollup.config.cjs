const path = require('path');

const createRollupConfig = require('../ds-dev-config/build/rollup-config-factory.cjs');

const srcDir = path.resolve(__dirname, `src`);
const pathToTsConfig = 'libs/ds-content/tsconfig.lib.json';
const pathToCSS = path.resolve(__dirname, `**/*.{css,scss}`);
const outputDir = path.resolve('./', `dist/libs/ds-content/`);

module.exports = (config) =>
  createRollupConfig(config, srcDir, pathToTsConfig, pathToCSS, outputDir);
