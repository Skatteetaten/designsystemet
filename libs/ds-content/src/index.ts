export * from './Card/Card';
export * from './Card/Card.types';
export * from './CardAlert/CardAlert.types';
export * from './DescriptionList/DescriptionList';
export * from './DescriptionList/DescriptionList.types';
export * from './Panel/Panel';
export * from './Panel/Panel.types';
