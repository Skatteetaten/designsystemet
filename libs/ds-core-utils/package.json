{
  "name": "@skatteetaten/ds-core-utils",
  "groupId": "no.skatteetaten.aurora",
  "version": "1.1.1",
  "homepage": "https://www.skatteetaten.no/stilogtone/designsystemet/",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Skatteetaten/designsystemet.git"
  },
  "publishConfig": {
    "registry": "https://nexus.sits.no/repository/npm-internal/"
  },
  "peerDependencies": {
    "i18next": "^23",
    "react": "^18",
    "react-i18next": "^14"
  },
  "scripts": {}
}
