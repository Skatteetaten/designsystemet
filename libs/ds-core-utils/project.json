{
  "name": "ds-core-utils",
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/ds-core-utils/src",
  "projectType": "library",
  "tags": [],
  "implicitDependencies": ["!ds-dev-config"],
  "targets": {
    "build": {
      "executor": "@nx/rollup:rollup",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/ds-core-utils",
        "tsConfig": "libs/ds-core-utils/tsconfig.lib.json",
        "project": "libs/ds-core-utils/package.json",
        "entryFile": "libs/ds-core-utils/src/index.ts",
        "external": ["react/jsx-runtime"],
        "rollupConfig": ["libs/ds-core-utils/rollup.config.cjs"],
        "compiler": "swc",
        "assets": [
          {
            "glob": "libs/ds-core-utils/README.md",
            "input": ".",
            "output": "."
          },
          {
            "glob": "LICENSE",
            "input": ".",
            "output": "."
          }
        ],
        "babelUpwardRootMode": true
      }
    },
    "lint": {
      "executor": "@nx/eslint:lint",
      "outputs": ["{options.outputFile}"]
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/libs/ds-core-utils"],
      "options": {
        "jestConfig": "libs/ds-core-utils/jest.config.ts"
      }
    },
    "stylelint": {
      "executor": "nx-stylelint:lint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": [
          "libs/ds-core-utils/**/*.css",
          "libs/ds-core-utils/**/*.scss"
        ]
      }
    },
    "tsc": {
      "executor": "@skatteetaten/ds-dev-config:tsCheck",
      "options": {
        "tsConfig": ["tsconfig.lib.json", "tsconfig.spec.json"]
      },
      "dependsOn": [
        {
          "projects": "ds-core-designtokens",
          "target": "build"
        }
      ]
    },
    "set-version": {
      "executor": "nx:run-commands",
      "options": {
        "command": "npm version {args.packageVersion} --no-git-tag",
        "cwd": "libs/ds-core-utils"
      }
    },
    "publish": {
      "executor": "nx:run-commands",
      "options": {
        "command": "npm publish dist/libs/ds-core-utils {args.switches}"
      }
    }
  }
}
