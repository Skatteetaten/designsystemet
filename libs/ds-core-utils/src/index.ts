export * from './SkatteetatenLogo/SkatteetatenLogo';
export * from './base-props.types';
export * from './defaults';
export * from './ExternalLayout/ExternalLayout';
export * from './InternalLayout/InternalLayout';
export * from './i18n';
export * from './Separator/Separator';
export * from './hooks/hooks';
export * from './formatter/formatter';
