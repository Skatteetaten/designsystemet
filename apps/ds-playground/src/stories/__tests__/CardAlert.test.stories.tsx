import { Meta, StoryFn, StoryObj } from '@storybook/react';
import { expect, userEvent, waitFor, within } from '@storybook/test';

import { Card, CardAlertVariant } from '@skatteetaten/ds-content';
import { statusArr } from '@skatteetaten/ds-core-utils';
import { SkattetrekkSVGpath } from '@skatteetaten/ds-icons';
import { Heading, Paragraph } from '@skatteetaten/ds-typography';

const meta = {
  component: Card.Alert,
  title: 'Tester/Card/Alert',
  argTypes: {
    // Baseprops
    ref: { table: { disable: true } },
    className: { table: { disable: true } },
    id: { table: { disable: true } },
    lang: { table: { disable: true } },
    'data-testid': { table: { disable: true } },
    // Props
    children: {
      table: { disable: true },
      control: 'text',
    },
    variant: {
      table: { disable: true },
      control: 'text',
    },
    title: {
      table: { disable: true },
      control: 'text',
    },
    titleAs: {
      table: { disable: true },
    },
    showAlert: {
      table: { disable: true },
      control: 'text',
    },
    svgPath: {
      table: { disable: true },
    },
    // Events
    onClose: {
      table: { disable: true },
    },
  },
} satisfies Meta<typeof Card.Alert>;
export default meta;
type Story = StoryObj<typeof meta>;

const defaultText =
  'Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.';

const defaultArgs = {
  children: defaultText,
  variant: statusArr[1],
  title: 'Merknad',
};

export const WithRef = {
  name: 'With Ref (FA1)',
  args: {
    ...defaultArgs,
    ref: (instance: HTMLDivElement | null): void => {
      if (instance) {
        instance.id = 'dummyIdForwardedFromRef';
      }
    },
  },
  argTypes: {
    ref: { table: { disable: false } },
  },
  parameters: {
    imageSnapshot: { disable: true },
  },
  play: async ({ canvasElement }): Promise<void> => {
    const canvas = within(canvasElement);
    const container = canvas.getAllByRole('generic')[1];
    await expect(container).toHaveAttribute('id', 'dummyIdForwardedFromRef');
  },
} satisfies Story;

export const WithAttributes = {
  name: 'With Attributes (FA2-5)',
  args: {
    ...defaultArgs,
    id: 'htmlId',
    className: 'dummyClassname',
    lang: 'en',
    'data-testid': '123ID',
  },
  argTypes: {
    id: { table: { disable: false } },
    className: { table: { disable: false } },
    lang: { table: { disable: false } },
    'data-testid': { table: { disable: false } },
  },
  play: async ({ canvasElement }): Promise<void> => {
    const canvas = within(canvasElement);
    const cardNote = canvas.getAllByRole('generic')[1];
    await expect(cardNote).toHaveAttribute('id', 'htmlId');
    await expect(cardNote).toHaveClass('dummyClassname');
    await expect(cardNote).toHaveAttribute('lang', 'en');
    await expect(cardNote).toHaveAttribute('data-testid', '123ID');
  },
} satisfies Story;

export const WithSvgPath = {
  name: 'With custom icon, children and title (A7)',
  args: {
    ...defaultArgs,
    svgPath: SkattetrekkSVGpath,
  },
  argTypes: {
    children: { table: { disable: false } },
  },
} satisfies Story;

const variantsArr: CardAlertVariant[] = ['info', 'warning', 'danger'];
const TemplateAllVariants: StoryFn<typeof Card> = (args) => (
  <Card>
    {variantsArr.map((variant, index) => {
      return (
        <Card.Alert
          title={defaultArgs.title}
          {...args}
          key={`card_${index}`}
          className={'bottomSpacingXL'}
          variant={variant}
        >
          <Card.Content>{args.children}</Card.Content>
        </Card.Alert>
      );
    })}
  </Card>
);

export const AllVariants = {
  render: TemplateAllVariants,
  name: 'All Variants (A7)',
  args: {
    ...defaultArgs,
  },
} satisfies Story;

export const WithOnClose = {
  name: 'With OnClose',
  args: {
    ...defaultArgs,
    children: (
      <>
        <Heading as={'h1'} level={4}>
          {'Avvist av kortutsteder'}
        </Heading>
        <Paragraph>
          {'Ta kontakt med kortutsteder for '}
          <a href={'#storybook-root'}>{'mer informasjon'}</a>
        </Paragraph>
      </>
    ),
  },
  parameters: {
    imageSnapshot: { disable: true },
  },
  play: async ({ canvasElement }): Promise<void> => {
    const canvas = within(canvasElement);
    const closeButton = canvas.getByRole('button');
    await userEvent.click(closeButton);
    await waitFor(() => {
      expect(
        canvas.queryByText('Avvist av kortutsteder')
      ).not.toBeInTheDocument();
    });
  },
} satisfies Story;
