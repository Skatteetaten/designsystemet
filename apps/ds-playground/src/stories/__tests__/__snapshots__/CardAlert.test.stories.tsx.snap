// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tester/Card/Alert AllVariants smoke-test 1`] = `
<div data-test-block="true">
  <div class="_card_nrlm6_5"
       data-color="white"
  >
    <div class="_cardAlert_e7y3b_1 bottomSpacingXL"
         data-variant="info"
    >
      <div class="_cardAlertHeadingContainer_e7y3b_57">
        <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _cardAlertIcon_e7y3b_23"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M21 3H3v18h18zm-7.75 5.4h-2.5V6h2.5zm0 9.6h-2.5v-7.2h2.5z">
          </path>
        </svg>
        <h3 class="_heading_12p5o_1 _heading_level5_12p5o_47">
          Merknad
        </h3>
        <button class="_button_19ayz_1 _button_medium_19ayz_42  _cardAlertCloseButton_e7y3b_50"
                data-brightness="default"
                type="button"
        >
          <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _icon_19ayz_26"
               viewbox="0 0 24 24"
               focusable="false"
               aria-labelledby="svgtitle-:r7:"
               aria-hidden="false"
               role="img"
          >
            <title id="svgtitle-:r7:">
              Lukk
            </title>
            <path d="M19 17.61L13.89 12.5L19 7.39L17.11 5.5L12 10.61L6.89 5.5L5 7.39L10.11 12.5L5 17.61L6.89 19.5L12 14.39L17.11 19.5L19 17.61Z">
            </path>
          </svg>
        </button>
      </div>
      <div class="_cardContent_48t19_1">
        <div>
          Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.
        </div>
      </div>
      <div class="_cardAlertArrow_e7y3b_29">
      </div>
    </div>
    <div class="_cardAlert_e7y3b_1 bottomSpacingXL"
         data-variant="warning"
    >
      <div class="_cardAlertHeadingContainer_e7y3b_57">
        <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _cardAlertIcon_e7y3b_23"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M13 14h-2v-4h2v4Zm0 4h-2v-2h2v2ZM1 21h22L12 2 1 21Z">
          </path>
        </svg>
        <h3 class="_heading_12p5o_1 _heading_level5_12p5o_47">
          Merknad
        </h3>
        <button class="_button_19ayz_1 _button_medium_19ayz_42  _cardAlertCloseButton_e7y3b_50"
                data-brightness="default"
                type="button"
        >
          <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _icon_19ayz_26"
               viewbox="0 0 24 24"
               focusable="false"
               aria-labelledby="svgtitle-:r9:"
               aria-hidden="false"
               role="img"
          >
            <title id="svgtitle-:r9:">
              Lukk
            </title>
            <path d="M19 17.61L13.89 12.5L19 7.39L17.11 5.5L12 10.61L6.89 5.5L5 7.39L10.11 12.5L5 17.61L6.89 19.5L12 14.39L17.11 19.5L19 17.61Z">
            </path>
          </svg>
        </button>
      </div>
      <div class="_cardContent_48t19_1">
        <div>
          Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.
        </div>
      </div>
      <div class="_cardAlertArrow_e7y3b_29">
      </div>
    </div>
    <div class="_cardAlert_e7y3b_1 bottomSpacingXL"
         data-variant="danger"
    >
      <div class="_cardAlertHeadingContainer_e7y3b_57">
        <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _cardAlertIcon_e7y3b_23"
             viewbox="0 0 24 24"
             focusable="false"
             aria-hidden="true"
             role="img"
        >
          <path d="M15.73 3H8.27L3 8.27v7.46L8.27 21h7.46L21 15.73V8.27zM13 16h-2v-2h2zm0-4h-2V8h2z">
          </path>
        </svg>
        <h3 class="_heading_12p5o_1 _heading_level5_12p5o_47">
          Merknad
        </h3>
        <button class="_button_19ayz_1 _button_medium_19ayz_42  _cardAlertCloseButton_e7y3b_50"
                data-brightness="default"
                type="button"
        >
          <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _icon_19ayz_26"
               viewbox="0 0 24 24"
               focusable="false"
               aria-labelledby="svgtitle-:rb:"
               aria-hidden="false"
               role="img"
          >
            <title id="svgtitle-:rb:">
              Lukk
            </title>
            <path d="M19 17.61L13.89 12.5L19 7.39L17.11 5.5L12 10.61L6.89 5.5L5 7.39L10.11 12.5L5 17.61L6.89 19.5L12 14.39L17.11 19.5L19 17.61Z">
            </path>
          </svg>
        </button>
      </div>
      <div class="_cardContent_48t19_1">
        <div>
          Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.
        </div>
      </div>
      <div class="_cardAlertArrow_e7y3b_29">
      </div>
    </div>
  </div>
</div>
`;

exports[`Tester/Card/Alert WithAttributes play-test 1`] = `
<div data-test-block="true">
  <div id="htmlId"
       class="_cardAlert_e7y3b_1 dummyClassname"
       lang="en"
       data-testid="123ID"
       data-variant="info"
  >
    <div class="_cardAlertHeadingContainer_e7y3b_57">
      <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _cardAlertIcon_e7y3b_23"
           viewbox="0 0 24 24"
           focusable="false"
           aria-hidden="true"
           role="img"
      >
        <path d="M21 3H3v18h18zm-7.75 5.4h-2.5V6h2.5zm0 9.6h-2.5v-7.2h2.5z">
        </path>
      </svg>
      <h3 class="_heading_12p5o_1 _heading_level5_12p5o_47">
        Merknad
      </h3>
      <button class="_button_19ayz_1 _button_medium_19ayz_42  _cardAlertCloseButton_e7y3b_50"
              data-brightness="default"
              type="button"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _icon_19ayz_26"
             viewbox="0 0 24 24"
             focusable="false"
             aria-labelledby="svgtitle-:r3:"
             aria-hidden="false"
             role="img"
        >
          <title id="svgtitle-:r3:">
            Lukk
          </title>
          <path d="M19 17.61L13.89 12.5L19 7.39L17.11 5.5L12 10.61L6.89 5.5L5 7.39L10.11 12.5L5 17.61L6.89 19.5L12 14.39L17.11 19.5L19 17.61Z">
          </path>
        </svg>
      </button>
    </div>
    Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.
    <div class="_cardAlertArrow_e7y3b_29">
    </div>
  </div>
</div>
`;

exports[`Tester/Card/Alert WithOnClose play-test 1`] = `
<div data-test-block="true">
</div>
`;

exports[`Tester/Card/Alert WithRef play-test 1`] = `
<div data-test-block="true">
  <div class="_cardAlert_e7y3b_1"
       data-variant="info"
       id="dummyIdForwardedFromRef"
  >
    <div class="_cardAlertHeadingContainer_e7y3b_57">
      <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _cardAlertIcon_e7y3b_23"
           viewbox="0 0 24 24"
           focusable="false"
           aria-hidden="true"
           role="img"
      >
        <path d="M21 3H3v18h18zm-7.75 5.4h-2.5V6h2.5zm0 9.6h-2.5v-7.2h2.5z">
        </path>
      </svg>
      <h3 class="_heading_12p5o_1 _heading_level5_12p5o_47">
        Merknad
      </h3>
      <button class="_button_19ayz_1 _button_medium_19ayz_42  _cardAlertCloseButton_e7y3b_50"
              data-brightness="default"
              type="button"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _icon_19ayz_26"
             viewbox="0 0 24 24"
             focusable="false"
             aria-labelledby="svgtitle-:r1:"
             aria-hidden="false"
             role="img"
        >
          <title id="svgtitle-:r1:">
            Lukk
          </title>
          <path d="M19 17.61L13.89 12.5L19 7.39L17.11 5.5L12 10.61L6.89 5.5L5 7.39L10.11 12.5L5 17.61L6.89 19.5L12 14.39L17.11 19.5L19 17.61Z">
          </path>
        </svg>
      </button>
    </div>
    Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.
    <div class="_cardAlertArrow_e7y3b_29">
    </div>
  </div>
</div>
`;

exports[`Tester/Card/Alert WithSvgPath smoke-test 1`] = `
<div data-test-block="true">
  <div class="_cardAlert_e7y3b_1"
       data-variant="info"
  >
    <div class="_cardAlertHeadingContainer_e7y3b_57">
      <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _cardAlertIcon_e7y3b_23"
           viewbox="0 0 24 24"
           focusable="false"
           aria-hidden="true"
           role="img"
      >
        <path d="M11.52 4.656a7.61 7.61 0 0 1 1.682.18l-2.788 9.301 8.614-3.568c.15.61.223 1.235.22 1.863a7.723 7.723 0 0 1-7.728 7.728c-4.272 0-7.776-3.456-7.776-7.728S7.2 4.656 11.52 4.656Zm4.058-.789-.074-.027c2.01.7 3.623 2.234 4.475 4.17l-6.442 2.669 2.041-6.812Z">
        </path>
      </svg>
      <h3 class="_heading_12p5o_1 _heading_level5_12p5o_47">
        Merknad
      </h3>
      <button class="_button_19ayz_1 _button_medium_19ayz_42  _cardAlertCloseButton_e7y3b_50"
              data-brightness="default"
              type="button"
      >
        <svg class="_systemIcon_dry7n_2 _systemIcon_medium_dry7n_10 _icon_19ayz_26"
             viewbox="0 0 24 24"
             focusable="false"
             aria-labelledby="svgtitle-:r5:"
             aria-hidden="false"
             role="img"
        >
          <title id="svgtitle-:r5:">
            Lukk
          </title>
          <path d="M19 17.61L13.89 12.5L19 7.39L17.11 5.5L12 10.61L6.89 5.5L5 7.39L10.11 12.5L5 17.61L6.89 19.5L12 14.39L17.11 19.5L19 17.61Z">
          </path>
        </svg>
      </button>
    </div>
    Gruppering av opplysninger i skjema, for eksempel inntekter, personer eller oppsummeringer.
    <div class="_cardAlertArrow_e7y3b_29">
    </div>
  </div>
</div>
`;
